src/components/mdx-bread-crumbs/MdxBreadCrumbs.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react';
+
+import { theme } from 'src/styles/constants/theme';
+import styled from 'styled-components';
+
+const StyledBreadcrumbs = styled.div`
+  font-size: 14px;
+  font-weight: 500;
+  padding: 0 16px;
+
+  > a,
+  span {
+    color: ${theme.gray800};
+    border-bottom: 0px solid #fff;
+    margin: 0 3px 0 5px;
+    :hover {
+      opacity: 0.9;
+    }
+  }
+  a:first-of-type {
+    margin-left: 0;
+  }
+  span:last-of-type {
+    color: ${theme.gray300};
+  }
```

Comment:
I see that in your stories example, you are using `div` for the slash, do you think that should be span? if it's not, when will `span` scenario happen?

Reply:
This is how it was setup on Documentation. Maybe this will help answer your question or provide another idea?

```
<StyledBreadcrumbs className="breadcrumbs">
      <Link to="/">Home</Link> <Slash>{' /'}</Slash>
      {pathnames.map((path, index) => {
        if (path === pathToSkip) {
          return null;
        }
        return (
          <span key={path}>
            <Link to={`/${formatPath(pathnames, index)}` as IInternalPathname}>
              {formatPathName(path)}
            </Link>{' '}
            <Slash>{' /'}</Slash>
          </span>
        );
      })}
      <span>{frontmatterTitle || pathTitle}</span>
    </StyledBreadcrumbs>
```

Reply:
Yes, using the span as wrapper for the Slash (`div`), I don't think it a good HTML format 🤔  If this's the case, then I have an idea. 
What do you think about wrapping all `Link` inside of a `span`. And then using the `pseudo selector :after` for the Slash instead of using div? Then you won't have `div` inside of `span`.